test/AtomicFAddExt.ll

```diff
@@ -0,0 +1,118 @@
+; RUN: llvm-as < %s > %t.bc
```

Comment:
Why not just
```suggestion
; RUN: llvm-as %s -o %t.bc
```

?

To me, it seems more readable

Reply:
Done; must've been an oblivious copy & paste :)